packages/chat-component-bindings/src/handlers/createHandlers.ts

```diff
@@ -115,7 +115,11 @@ export const createDefaultChatHandlers = memoizeOne(
       },
       /* @conditional-compile-remove(rich-text-editor-image-upload) */
       onDeleteImage: async function (imageId: string): Promise<void> {
-        await chatThreadClient.deleteImage(imageId);
+        try {
```

Comment:
Is there a reason we have a try/catch with logging specifically for this call on the chatThreadClient? Where the rest of the calls in the file do not?

Reply:
> Is there a reason we have a try/catch with logging specifically for this call on the chatThreadClient? Where the rest of the calls in the file do not?

I'll create a KTLO ticket to investigate/update other functions.